hxrsnd/state.py

```diff
@@ -2,8 +2,7 @@
 from copy import copy
 
 from ophyd.device import Device, ComponentMeta
-from super_state_machine.machines import StateMachine
-from super_state_machine.extras  import PropertyMachine
+from super_state_machine.extras import PropertyMachine
```

Comment:
This whole file can probably be deprecated. Not used anywhere I don't think.

Reply:
Oh, good catch! I'll poke around a bit.

Reply:
Maybe we just let it sit for now, and if we delete it do it in a different PR